src/pages/search.vue

```diff
@@ -69,12 +79,8 @@ const BrowsePage = {
       local.get(process.env.filterStorageKey)
         ? local.get(process.env.filterStorageKey) === 'true'
         : true
-
-    const MIN_SCREEN_WIDTH_FILTER_VISIBLE_BY_DEFAULT = 768
-    const isDesktop = () =>
-      screenWidth() > MIN_SCREEN_WIDTH_FILTER_VISIBLE_BY_DEFAULT
     this.setFilterVisibility({
-      isFilterVisible: isDesktop() ? localFilterState() : false,
+      isFilterVisible: this.isLargeScreen ? localFilterState() : false,
```

Comment:
Can be simplified as
```suggestion
      isFilterVisible: this.isLargeScreen && localFilterState(),
```